multi_render.py

```diff
@@ -56,6 +56,13 @@ def quit():
 WIDTH = 1366
 KEYSIZE = min((WIDTH-1024)/5, HEIGHT / len(replays))
 
+# Only works with Height of 768 and Width of 1366 if you want different size you will have to work out yourself
+# This Change Centers the play on the window just like in Osu client
+# Helps with easy overlay for video
+X_CHANGE = 273
+Y_CHANGE = 89
+SCALE = 1.551
```

Comment:
This is worked out with the pixel difference between the play zone and the outer edge of the game 
![2016-08-31_22-26-03](https://cloud.githubusercontent.com/assets/11075927/18147018/b94dfa4a-6fca-11e6-9a30-cb1b6f2c4971.png)
The Scale is done by getting actual pixel of play zone (for 1366 its x = 794) and divide by in-game x so 512 

794/512 = 1.551